### ISSUE TITLE
Questions about the unchanged learning rate

### ISSUE THREAD

**Issue by akb46mayu** [zylo117/Yet-Another-EfficientDet-Pytorch#309]
Hi all,
Thanks for the great work on effdet. I was trying to train a model using your pretrained weights on my dataset but found the learning rate was unchanged after 30 epochs. The training loss (blue) was small enough, but the validation loss (magenta) was still larger and did not drop too much. Was that because of the usage of ReduceLROnPlateau scheduler? What If I use any stepwise learning rate decay scheduler like StepLR?
![lr_problem](https://user-images.githubusercontent.com/18405025/82152193-a2c52200-982d-11ea-9b23-15f4fa41f279.PNG)

**Comment by zylo117** [comment]
if lr was reduced on plateau, it should drop, but obviously, it didn't.
so it is simply overfitting, you should try lower lr.

**Comment by akb46mayu** [comment]
Thank you for your comments. The LR may not be low enough. I will try it.

**Comment by akb46mayu** [comment]
Just as zylo117 pointed out in another ticket, LR only drops on plateaus. Since no plateaus were detected by computing the mean training loss,  LR would not change. Thanks
